sdk/browser/README.md

```diff
@@ -103,8 +103,8 @@ Behaves the same as the wallet-core-sdk `confirmSignIn` method, but with the add
 #### Confirm sign in (if using confirmSignIn for both sign up and login scenarios)
 
 ```ts
-const issueSignupCredential = true
-const { isNew, commonNetworkMember: affinityWallet } = await AffinityWallet.confirmSignIn(token, confirmationCode, options, issueSignupCredential)
+const options = {issueSignupCredential: true  }
```

Comment:
@Volland ideally at least apiKey and env should be passed here as well, so better probably mention that its extension by `issueSignupCredential` general options.